frontend/src/components/commons/Notepad.jsx

```diff
@@ -1,102 +1,180 @@
 import React, { Component } from "react";
+import PropTypes from "prop-types";
 import LOCALIZE from "../../text_resources";
 import TextareaAutosize from "react-textarea-autosize";
-import { BROWSER_STRING, IE_STRING, VALID_BROWSERS } from "../../Status";
 import "../../css/emib-tabs.css";
 
 const styles = {
   windowPadding: {
     paddingTop: 43,
-    paddingLeft: 20,
     order: 2
   },
   h4: {
-    color: "white"
+    textAlign: "left",
+    fontSize: "16px",
+    fontWeight: "bold",
+    color: "#00565e",
+    padding: "0 0 8px 12px",
+    borderBottom: "1.5px solid #88C800"
   },
-  title: {
-    paddingLeft: 12,
-    paddingTop: 0.1,
-    height: 55,
-    backgroundColor: "#00565e",
-    borderRadius: "5px 5px 0 0"
+  hideNotepadBtn: {
+    float: "right",
+    margin: "8px 12px 0 0",
+    padding: "0 8px 0 8px",
+    backgroundColor: "transparent",
+    border: "none",
+    cursor: "pointer"
+  },
+  hideNotepadBtnIcon: {
+    paddingTop: 12,
+    cursor: "pointer"
+  },
+  hideNotepadBtnZone: {
+    float: "right"
   },
   content: {
     backgroundColor: "white",
-    borderWidth: "1px 1px 0 1px",
+    borderWidth: "1px 1px 0 3px",
     borderStyle: "solid",
     borderColor: "#00565e",
-    borderRadius: "5px 5px 0 0",
-    width: 220,
-    height: "calc(100vh - 237px)"
+    borderRadius: "0 5px 0 0",
+    width: "100%",
+    height: "calc(100vh - 238px)"
   },
   notepadSection: {
     overflow: "auto",
-    height: "calc(100vh - 297px)"
+    height: "calc(100vh - 291px)"
   },
   textArea: {
+    padding: "0 12px 6px 12px",
+    width: "100%",
     resize: "none",
     border: "none"
   },
-  center: {
-    textAlign: "center"
+  openNotepadBtnLabel: {
+    position: "absolute",
+    paddingTop: 40,
+    width: 60,
+    fontSize: "13px",
+    color: "white",
+    cursor: "pointer"
+  },
+  openNotepadBtn: {
+    width: 60,
+    border: "none",
+    backgroundColor: "#00565e",
+    height: "calc(100vh - 238px)",
+    cursor: "pointer",
+    borderRadius: "0 5px 0 0"
+  },
+  openNotepadBtnIcon: {
+    position: "absolute",
+    padding: "16px 0 0 22px",
+    cursor: "pointer",
+    color: "white"
+  },
+  openNotepadBtnHeight: {
+    height: "calc(100vh - 238px)"
   }
 };
 
 class Notepad extends Component {
+  static propTypes = {
+    notepadContent: PropTypes.string
+  };
+
   state = {
-    columnWidth: 0
+    notepadHidden: false,
+    notepadContent: ""
   };
 
-  /*
-  Adjust the notepad text zone width depending on the browser
-  Each browser has its own interpretation of the notepad text zone width
-  */
-  detectBrowser = () => {
-    switch (BROWSER_STRING) {
-      case IE_STRING:
-        this.setState({ columnWidth: 20 });
+  handleCollapse = arg => {
+    switch (arg) {
+      //When Closing the Notepad
+      case "handleHide":
+        this.setState({ notepadHidden: true });
+        //Get current notepad content
+        let x = document.getElementById("notepad-content");
```

Comment:
Instead of using `defaultValue` and finding the element by id here and updating the state only when changing the notepad, you can always store the `value` in state `notepadContent` like this: https://codepen.io/gaearon/pen/VmmPgp?editors=0010

Reply:
Fixed in commit #https://github.com/code-for-canada/project-thundercat/pull/150/commits/ef911b4c86bdd6dab57521372ff853cb20ee697b

Reply:
Fixed in commit #https://github.com/code-for-canada/project-thundercat/pull/150/commits/62240a23da36afb00bce1751f6b0008ad3a4ab81